docs/en/siem/cases-api-add-comment.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[cases-api-add-comment]]
+=== Add comment
+
+Adds a comment to an existing case.
+
+==== Request URL
+
+`POST <kibana host>/<port>/api/cases/<case ID>/comments`
+
+===== URL parts
+
+The URL must include the `case ID` of the case to which you are adding a 
+comment. Call <<cases-api-find-cases>> to retrieve case IDs.
+
+==== Request body
+
+A JSON object with a `comment` field:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comment` |String |The case's new comment. |Yes
+|==============================================
+
+===== Example request
+
+Adds a comment to case ID `a18b38a0-71b0-11ea-a0b2-c51ea50a58e2`:
+
+[source,sh]
+--------------------------------------------------
+POST api/cases/a18b38a0-71b0-11ea-a0b2-c51ea50a58e2/comments
+{
+  "comment": "That is nothing - Ethan Hunt answered a targeted social
+  media campaign promoting phishy pension schemes to IMF operatives."
+}
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+   Indicates a successful call.
+
+==== Response payload
+
+A JSON object that includes the user who created the comment, and the comment's
+ID, version, and creation time.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
```

Comment:
related to that comment https://github.com/elastic/stack-docs/pull/761#discussion_r402361161